app/controllers/mainstream_browse_pages_controller.rb

```diff
@@ -1,46 +1,84 @@
 class MainstreamBrowsePagesController < ApplicationController
-  include TagCreateUpdatePublish
-  tag_create_update_publish_for MainstreamBrowsePage
-
   before_filter :require_gds_editor_permissions!
 
-  def edit
-    set_topics_for_select
+  def index
+    @pages = MainstreamBrowsePage.sorted_parents
```

Comment:
We're using `@pages` here, and `@browse_page` in other actions.  Would be good to make these consistent.